scripts/generate-types-package.ts

```diff
@@ -0,0 +1,78 @@
+import {copyFile, readJson, writeFile, writeJSON } from 'fs-extra'
+import { resolve } from 'path'
+import { cwd } from 'process'
+import { execSync } from 'child_process'
+import depcheck, { PackageDependencies } from 'depcheck'
+
+run()
+    .then(() => process.exit(0))
```

Comment:
2 spaces for indentation please :)

Reply:
Sure, will do. I noticed `eslint` doesn't fix this special case since the rule option `memberExpression` is set to to `off` in the `.eslintrc.json`:

> `"MemberExpression"` (default: 1) enforces indentation level for multi-line property chains. This can also be set to `"off"` to disable checking for MemberExpression indentation.
> https://eslint.org/docs/rules/indent

Shall I switch this option to `2` spaces instead?

Reply:
I think I disabled it so we can choose between these two cases:

```
run().then(...
       .catch(
```

Or 

```
longVariableName.subVariableName.doSomethingInteresting()
  .then(...
```

Reply:
Just made a few tests with `"MemberExpression": 1`:
```
run()
    .then(() => process.exit(0)).catch(err => {
    console.error(err)
    process.exit(-1)
  })

depcheck.detector.requireCallExpression
    .toString()
```
`yarn eslint --fix scripts/generate-types-package.ts` ->
```
run()
  .then(() => process.exit(0)).catch(err => {
    console.error(err)
    process.exit(-1)
  })

depcheck.detector.requireCallExpression
  .toString()
```
It seems it doesn't "break" the expression but just fix the number of space used for indentation.
I'd usually recommend to automate any code style which can be automated, even if it means there would be a side effect. 

The side effect is usually deem acceptable. But up to you! I could submit this suggestion in another PR I guess.

Reply:
> The side effect is usually deem acceptable. But up to you! I could submit this suggestion in another PR I guess.

yes please :+1: